test/terra/backends/qasm_simulator/qasm_measure.py

```diff
@@ -129,3 +129,15 @@ def test_measure_nondeterministic_multi_qubit_without_sampling(self):
             qobj, backend_options=self.BACKEND_OPTS).result()
         self.is_completed(result)
         self.compare_counts(result, circuits, targets, delta=0.05 * shots)
+
+    def test_measure_conditional(self):
```

Comment:
Can you remove this test case since (for now) conditional measure shouldn't be allowed by the qobj spec